large_image/report.py

```diff
@@ -0,0 +1,65 @@
+import scooby
+
+
+class Report(scooby.Report):
+    def __init__(self, additional=None, ncol=3, text_width=80, sort=False):
+        """Initiate a scooby.Report instance."""
+        # Mandatory packages.
+        core = [
+            'large_image',
+            'cachetools',
+            'PIL',
+            'psutil',
+            'numpy',
+            'palettable',
+            'scooby',
+        ]
+        # Optional packages.
+        girder = [
+            'girder_large_image',
```

Comment:
I'm not following the logic of which packages are explicitly listed versus just using scooby to track imports as they happen.  Shouldn't this have `girder_large_image_annotation`, too?  Why include `girder_worker_utils` as it is a dependency that will get pulled in by girder_worker and isn't explicitly asked for.